src/routes/groupRoutes.js

```diff
@@ -84,6 +84,7 @@ async function handleDeleteGroup(req, res, next) {
     const id = req.params.id;
     await groups.destroy({ where: { id } });
     await associations.destroy({ where: { groupId:id } });
+    await santaPairs.destroy({ where: { groupId:id } });
```

Comment:
when a group is destroyed, it will also destroy associations tied to that groupId in the santapairs table